SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -70,17 +84,17 @@ public final class SwiftGenStoryboardEnumBuilder {
         }
     }
     
-    public func build(enumName enumName: String? = "Name", indentation indent : SwiftGenIndentation = .Spaces(4)) -> String {
+    public func build(scenesEnumName scenesEnumName: String = "Scene", seguesEnumName: String = "Segue", indentation indent : SwiftGenIndentation = .Spaces(4)) -> String {
         var text = "// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen\n\n"
         let t = indent.string
         text += commonCode(indentationString: t)
 
         text += "extension UIStoryboard {\n"
-        let s = enumName == nil ? "" : t
-        if let enumName = enumName {
-            text += "\(t)enum \(enumName.asSwiftIdentifier()) {\n"
-        }
-        for (name, scenes) in storyboards {
+      
+        /// Scenes
+        let s = t
```

Comment:
Now that those params are not optional anymore and those intermediate enums can't be omitted, this `s` can go away and be replaced by `t` everywhere